test/integration/running/test_running.py

```diff
@@ -190,6 +191,7 @@ def test_clean(tt_cmd, tmpdir_with_cfg):
     stop_rc, stop_out = run_command_and_get_output(stop_cmd, cwd=tmpdir)
     assert stop_rc == 0
     assert re.search(r"The Instance test_app \(PID = \d+\) has been terminated.", stop_out)
+    time.sleep(5)
```

Comment:
Why do we need to add a timeout here?

Reply:
Sometimes app on ci doesn't stop in time after log message. Probably somehow related with #264. Will wait for fix and rebase.